myuw/test/util/test_settings.py

```diff
@@ -2,7 +2,9 @@
 from django.conf import settings
 from myuw.util.settings import get_calendar_time_zone,\
     get_mailman_courserequest_recipient, get_google_search_key,\
-    get_legacy_url, get_logout_url, get_myuwclass_url
+    get_legacy_url, get_logout_url, get_myuwclass_url,\
+    get_myuw_admin_group, get_myuw_override_group, get_myuw_astra_group_stem,\
+    get_disable_actions_when_override
 
 
 class TestSetting(TestCase):
```

Comment:
Same here as above:
This test doesn't pass on my build, could you override the settings required on the individual test case level?

Reply:
Sure!

Reply:
Sorry, I didn't get it earlier. Thanks for clarifying!
Updated.